docs/step-by-step/kubeapps-on-tkg/step-2.md

```diff
@@ -35,41 +31,50 @@ kubeops: # main backend APIs configuration
 
 assetsvc: # asset service configuration
 
-frontend: # nginx configuration (used to serve the dashboard and proxying the users' requests to the backend)
+frontend: # NGINX configuration (used to serve the dashboard and proxying user requests to the backend)
 
-ingress: # if you have an ingress controller installed on your cluster, you can use it to expose Kubeapps.
+ingress: # ingress controller details to expose Kubeapps.
 
 postgresql: # database configuration
-
-# other top-level entries, but not likely to be modified
 ```
 
-In this step, we focus on two main categories of the [values.yaml](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/values.yaml) file: authentication and look and feel.
+> **TIP**: Refer to the [Bitnami Kubeapps Helm chart documentation](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/README.md) for more information.
+
+The two main configuration areas are authentication and user interface.
 
-- **Authentication** includes: i) `clusters` to define the list of clusters that Kubeapps can target and which of them will use Pinniped; ii) `pinnipedProxy`: to enable the _Pinniped Proxy_ component; iii) `authProxy`: to define the flags used by _OAuth2 Proxy_, the component for performing the actual OIDC login.
-- **Look and feel** comprises: i) `dashboard.customStyle` for injecting custom CSS; ii) `dashboard.customLocale` for customizing some supported strings; iii) `apprepository.initialRepos` for defining the repositories included by default during the installation.
+- Key authentication parameters are:
 
-> **TIP**: these values can be entered in two different ways:
+i) `clusters` to define the list of clusters that Kubeapps can target and which of them will use Pinniped;
+ii) `pinnipedProxy` to enable the _Pinniped Proxy_ component;
```

Comment:
It's not clear to me what rules you followed when formatting as code or wiht underscore. Can you clarify? I'm not sure why Pinniped Proxy is formatted this way.

Reply:
Any formal rule, just trying to have the most technical names with a different format (separated from `xxxxx` meaning something in the code)